src/js/settings.js

```diff
@@ -506,7 +506,61 @@ function importSubscriptions(){
     })
   });
 }
+/**
+* Import NewPipe Channels Subscriptions
+* @return {Void}
+*/
+function importNewpipeSubscriptions(){
+  // Open user's file browser.  Only show .json files.
+ dialog.showOpenDialog({
+  properties: ['openFile'],
+  filters: [
+    {name: 'JSON', extensions: ['*']},
+  ]
+}, function(fileLocation){
+  if(typeof(fileLocation) === 'undefined'){
+    console.log('Import Aborted');
+    return;
+  }
+  console.log(fileLocation);
+  let i = fileLocation[0].lastIndexOf('.');
+  let fileType = (i < 0) ? '' : fileLocation[0].substr(i);
+  console.log(fileType);
+
+  fs.readFile(fileLocation[0], function(readErr, data){
+    if(readErr){
+      showToast('Unable to read file.  File may be corrupt or have invalid permissions.');
+      throw readErr;
+    }
 
+    if (data.includes("app_version")){
```

Comment:
Add this check to `importSubscriptions()` instead and make a call to this function.